src/Home/ProximityTracingActivationStatus.tsx

```diff
@@ -39,9 +46,16 @@ export const ProximityTracingActivationStatus: FunctionComponent = () => {
     )
   }
 
-  const handleOnPressFix = () => {
-    exposureNotifications.request()
-    if (isNotAuthorized) {
+  const handleOnPressFix = async () => {
+    try {
+      await exposureNotifications.request()
+
+      const isNotENEnabled = status !== ENPermissionStatus.ENABLED
+
+      if (isNotENEnabled) {
```

Comment:
This is a rare case where I actually find not naming to be more clear:

```suggestion
      if (status !== ENPermissionStatus.ENABLED) {
```

Maybe it would be optimal if we renamed `status` to `enStatus` or something. 🤔